README.md

```diff
@@ -14,23 +14,29 @@ dynamic modules completely in Rust.
 
 In short, this SDK allows writing NGINX modules using the Rust language.
 
+It contains the following Rust crates:
+ * [nginx-sys](./nginx-sys) - allows to use ngx_* C functions via FFI when implementing modules. The `-sys` is used to follow a [naming convention](https://doc.rust-lang.org/cargo/reference/build-scripts.html#-sys-packages) to link with a C library.
+ * [ngx](./src) - it an opinionated SDK to make it a bit easer to use [nginx-sys](./nginx-sys) crate. Implements `macro_rules`, provides a way to build a dynamic module without any C code (see `ngx_modules!` macro_rule).
+
 ## Build
 
 NGINX modules can be built against a particular version of NGINX. The following environment variables can be used to specify a particular version of NGINX or an NGINX dependency:
 
-* `ZLIB_VERSION` (default 1.2.13) -
+* `ZLIB_VERSION` (default 1.3)
 * `PCRE2_VERSION` (default 10.42)
 * `OPENSSL_VERSION` (default 3.0.7)
 * `NGX_VERSION` (default 1.23.3) - NGINX OSS version
 * `NGX_DEBUG` (default to false)-  if set to true, then will compile NGINX `--with-debug` option
+* `NGX_SRC_DIR` (default not set) - When the value is set, then use this NGINX source code folder to build bindings from
+* `NGX_CONFIGURE_ARGS` (default not set) - When the value is set, then run the NGINX configure script with
```

Comment:
Suggest: When the value is set, use these arguments with the NGINX configure script